pkg/shell/hosts_dialog.jsx

```diff
@@ -278,7 +280,10 @@ class AddMachine extends React.Component {
                        </Button>
                    </>}
             >
-                {body}
+                <Stack hasGutter>
+                    { this.props.dialogError && <ModalError dialogError={this.props.dialogError} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221203-085317-c9401003-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)